spec/ADAS/ADAS.vspec

```diff
@@ -1,11 +1,7 @@
-#
-# (C) 2022 Robert Bosch GmbH
-# (C) 2018 Volvo Cars
-# (C) 2016 Jaguar Land Rover
+## Copyright (c) 2016 Contributors to COVESA
```

Comment:
Alway choosing the oldest, is that by intention, or should it be newest or "oldest"-"last touched"

Reply:
Yes, reusing what Eclipse writes at https://www.eclipse.org/projects/handbook/#ip-copyright-headers. But I am open to any policy

_The {year} is the year in which the content was created (e.g. “2004”)._

_The year is expressed either as a single year or a range spanning the year of the initial creation and that of the most recent change._



_Do we need to specify a range of years in the copyright statement?_

_No. In the past, legal advice was that the year of the initial creation of the content and the year of the last change should be reflected in the copyright header. This is no longer the case. Specify the year that the content was initially created in the copyright statement._

_Do I need to update the year in the copyright statement when I make changes to the file?_

_No. The year in the copyright header should be the year that the content was created._